pages/calendar/components/Calendar/CalendarHeader/index.tsx

```diff
@@ -11,11 +11,12 @@ import CalendarTypeSegmentedButton from "../../CalendarTypeSegmentedButton";
 const cn = classNames.bind(styles);
 
 interface CalendarHeader {
```

Comment:
```suggestion
interface CalendarHeaderProps {
```
p4: 저희 팀 컨벤션에 맞게 `CalendarHeaderProps` 이름을 변경해도 좋을 것 같아요!